auraescript/src/bin/main.rs

```diff
@@ -65,6 +65,9 @@ fn main() -> anyhow::Result<()> {
 
     let _ = js_runtime.execute_script("", "Deno.core.initializeAsyncOps();")?;
 
+    // include helper.ts
+    let _ = js_runtime.execute_script("", include_str!("../../helpers.js"))?;
```

Comment:
if this is now some sort of core runtime perhaps it should be called "core.js" or "aurae.js" or something?

Reply:
I like aurae.js.
I'll try to keep it as typescript and compile it to js in build.rs

Reply:
Renamed to aurae.js